doc/architecture/repos.md

```diff
@@ -2,29 +2,47 @@
 
 This page is a central point for documenting the Ocean GitHub repositories in use.
 
-## Frontend
-
-* [Ocean Pleuston Frontend](https://github.com/oceanprotocol/pleuston/)
+## Pleuston
+* [Ocean Frontend Pleuston](https://github.com/oceanprotocol/pleuston/)
 
 ## Keeper
+* [Ocean Keeper Contracts](https://github.com/oceanprotocol/keeper-contracts/)
 
-* [Keeper Contracts](https://github.com/oceanprotocol/keeper-contracts/)
-
-## Provider
+## Aquarius
+* [Ocean Backend Aquarius](https://github.com/oceanprotocol/aquarius)
 
-* [Ocean Provider](https://github.com/oceanprotocol/provider)
+## Nautilina
```

Comment:
@MarcusJones said that repo probably won't be getting updated. Apparently it was created for the Plankton release. I'm not sure what will replace it, maybe [oceanprotocol/mantaray](https://github.com/oceanprotocol/mantaray).

Reply:
removed, thanks for the hint

Reply:
Yes, confirmed that Mantaray replaces Nautilina